desktopApp/app.js

```diff
@@ -0,0 +1,142 @@
+// Modules to control application life and create native browser window
```

Comment:
Did you tried to change this file to Typescript?

Reply:
just ignore this file i am gonna delete this i have converted it in ts as index.ts